### DISCUSSION TITLE
Pull Quarters from pocket instead of hand.

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#54] [Ideas]
Would be more natural to pull Quarters from Pant pocket instead of from hand.

**Comment by WilsonBradley** [comment]
Instead have watch on hand that will bring up menu/controls/settings

**Comment by curif** [comment]
I've been considering in something more "physical" for the configuration. Some kind of control panel mounted to the wall in the workshop, or in the rooms. And configure controls, volume, etc using buttons and handlers.
We are in a simulated VR environment, IMHO switch to a 2D screen for configuration breaks the simulation.
![image](https://user-images.githubusercontent.com/8068749/211830626-c6ab3216-fb34-42b5-a872-1fe5908d5450.png)

**Comment by WilsonBradley** [comment]
This is a great IDEA.
Player goes into a Configuration area to make changes.
.. Thanks..
